refs/wg21.json

```diff
@@ -20370,7 +20370,7 @@
         "source": "https://wg21.link/index.json"
     },
     "CWG435": {
-        "title": "Change \"declararation or definition\" to \"declaration\"",
+        "title": "Change \"declaration or definition\" to \"declaration\"",
```

Comment:
That seems possibly a correction of a correction

Reply:
I mostly tried to skip those. But at some point I was letting Google Sheets make suggestions and guessing they were ok. Will drop.